test/OAuth2ManagerTests/OAuth2APITests.h

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#pragma once
+
+#include <cassert>
+#include <string_view>
+
+
+// The 'client_id' describes the behavior and expectations of our mocked authorization server
+// Specifying grant type is required
+#define GRANT_TYPE_CODE L"grant=code"
```

Comment:
What is the reasoning some contants are written as #define marco and some are written in using inline constexpr

Reply:
**#define** directive used to define macros which is typically used for simple constant values or expressions that do not have a specific type while **constexpr** allows the compiler to perform constant expressions evaluation at compile time, ensuring type safety and better integration with the type system. 
So, for simple constant values #define macro is used and for expressions constexpr is used.